internal/term/raw.go

```diff
@@ -25,5 +25,5 @@ func SetRaw(fd int) error {
 	n.Cc[syscall.VMIN] = 1
```

Comment:
change go module name to `openGemini/go-prompt`

Reply:
Done

> change go module name to `openGemini/go-prompt`